pkg/service/backup/worker_manifest.go

```diff
@@ -71,8 +71,8 @@ func (w *worker) createTemporaryManifest(h hostInfo, tokens []int64) ManifestInf
 		},
 		Index: make([]FilesMeta, len(dirs)),
 	}
-	if w.Schema != nil {
-		c.Schema = RemoteSchemaFile(w.ClusterID, w.TaskID, w.SnapshotTag)
+	if w.Schema != nil && w.SchemaFilePath != "" {
```

Comment:
Is this added check necessary ?
`c.Schema` is pre-initalized somehow and cannot be updated to "" ?

Reply:
It shouldn't be, those two values should be always both initialized of uninitialized.
On the other hand, this is some form of another safety measure that doesn't harm anything, so I would be fine with leaving it as it is.